src/index.js

```diff
@@ -1 +1,73 @@
 'use strict';
+
+const fs = require('fs');
+const http = require('http');
+
+const PORT = 8000;
+
+const server = new http.Server();
+
+server.on('request', async(request, response) => {
+  // Get Styles:
+  if (request.url === '/src/public/main.css' && request.method === 'GET') {
+    const styles = fs.readFileSync('src/public/main.css');
+
+    response.setHeader('Content-Type', 'text/css');
+    response.end(styles);
+
+    return;
+  }
+
+  // Get all expenses:
+  if (request.url === '/expense' && request.method === 'GET') {
+    response.end(fs.readFileSync('src/data.json', 'utf-8'));
+  }
+
+  // Get index.html:
+  if (request.url === '/' && request.method === 'GET') {
+    const file = fs.createReadStream('src/public/index.html');
+
+    response.setHeader('Content-Type', 'text/html');
+    file.pipe(response);
+  }
```

Comment:
it's an additional functionality, right? just in case if you need to get the list of all expenses or be returned to the main page